mtprotosender.py:228 #INFO     [2025-09-25 23:40:32,563] - telethon.network.mtprotosender - 0| Connecting to 149.154.167.51:443/TcpFull...
mtprotosender.py:279 #INFO     [2025-09-25 23:40:32,604] - telethon.network.mtprotosender - 0| Connection to 149.154.167.51:443/TcpFull complete!
handlers.py:167 #INFO     [2025-09-25 23:40:35,544] - root - 0| Handler called. MessageDeleted
updates.py:443 #INFO     [2025-09-25 23:41:03,316] - telethon.client.updates - 0| Got difference for channel 1968379726 updates
handlers.py:167 #INFO     [2025-09-25 23:41:03,317] - root - 0| Handler called. MessageDeleted
handlers.py:167 #INFO     [2025-09-25 23:41:07,671] - root - 0| Handler called. MessageDeleted
mtprotosender.py:319 #INFO     [2025-09-25 23:41:21,206] - telethon.network.mtprotosender - 0| Disconnecting from 149.154.167.51:443/TcpFull...
mtprotosender.py:339 #INFO     [2025-09-25 23:41:21,207] - telethon.network.mtprotosender - 0| Disconnection from 149.154.167.51:443/TcpFull complete!
base_events.py:1758 #ERROR    [2025-09-25 23:41:21,253] - asyncio - 0| Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x0000027355F7A4D0>
